include/eggs/variant/bad_variant_access.hpp

```diff
@@ -42,7 +42,11 @@ namespace eggs { namespace variants
         template <typename T>
         EGGS_CXX11_NORETURN inline T throw_bad_variant_access()
         {
+#ifndef EGGS_VARIANT_NO_EXCEPTIONS
             throw bad_variant_access{};
+#else
+            std::terminate();
```

Comment:
`std::terminate` is defined in `<exception>`, which is not included in this header. Please fix.

(While you are up there feel free to remove the include for `<string>`, a leftover from 6fb973512b1af5af783bdae4c7bf9361e937eb4b)